src/py/mat3ra/made/tools/build/perturbation/configuration.py

```diff
@@ -0,0 +1,26 @@
+from typing import Callable, Dict, Tuple
+
+from mat3ra.code.entity import InMemoryEntity
+from mat3ra.made.material import Material
+from pydantic import BaseModel
+
+from ...utils import PerturbationFunctionHolder
+
+
+class PerturbationConfiguration(BaseModel, InMemoryEntity):
+    material: Material
+    perturbation_function: Tuple[Callable, Dict] = PerturbationFunctionHolder.sine_wave()
```

Comment:
We need to call things for what they are if this is a tuple, we should call it a tuple, or object, or `function_config_tuple`

Reply:
I restructured instead